doc/source/cluster/kubernetes/k8s-ecosystem/prometheus-grafana.md

```diff
@@ -45,8 +45,48 @@ kubectl get all -n prometheus-system
         enabled: true
         org_role: Viewer
   ```
-
-
+* In my environment (with Ray version 2.9.0+), the Ray dashboard encountered a cross-origin issue when embedding Grafana charts. This problem was resolved after modifying the grafana.ini configuration file. Below is the complete configuration file:
```

Comment:
let's phrase this slightly differently:

Something like:
If your browser is throwing cross-origin errors and blocking embeds from different origins, you can configure Grafana to allow embeds from specific origins.

Reply:
Let's also only include the configurations that matter, "allowed_origins" and possibly "cookie_samesite"?

Reply:
Thanks, your description is more accurate. I am currently only using kuberay in k8s, and cookie_samesite is not needed for the time being.